odxtools/diaglayers/hierarchyelement.py

```diff
@@ -31,7 +31,7 @@
 from .hierarchyelementraw import HierarchyElementRaw
 
 if TYPE_CHECKING:
-    from .database import Database
+    from ..database import Database
```

Comment:
mypy didn't catch this ?

Reply:
@zariiii9003: how did you find this? if using tools, can/should we change the CI system to do the same?

Reply:
I noticed red squiggly lines in my IDE. I also wonder why both ruff and mypy seem to ignore it

Reply:
ok. Seems like you should continue to let your IDE look at odxtools. mine (emacs) is not sophisticated enough ;)